fenix/app/src/main/res/layout/tracking_protection_learn_more_preference.xml

```diff
@@ -19,69 +19,45 @@
         app:layout_constraintTop_toTopOf="parent"
         app:srcCompat="@drawable/ic_etp_artwork" />
 
-    <TextView
-        android:id="@android:id/title"
-        android:layout_width="0dp"
-        android:layout_height="wrap_content"
-        android:layout_marginStart="20dp"
-        android:paddingTop="20dp"
-        android:ellipsize="none"
-        android:letterSpacing="0.05"
-        android:scrollHorizontally="false"
-        android:singleLine="false"
-        android:text="@string/preference_enhanced_tracking_protection_explanation_title"
-        android:textAlignment="viewStart"
-        android:textAppearance="@style/Header16TextStyle"
-        android:textColor="@color/photonLightGrey05"
-        app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintEnd_toStartOf="@id/guideline"
-        app:layout_constraintStart_toStartOf="parent" />
-
     <TextView
         android:id="@android:id/summary"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
-        android:paddingTop="12dp"
-        android:paddingBottom="12dp"
+        android:layout_marginStart="16dp"
+        android:layout_marginTop="16dp"
```

Comment:
checking this while, work in progress

Reply:
Yes, the provided image is actually slightly different to the Figma design. I've cc'd you into some UX conversations around this ETP banner, which should provide some context 🙂